Gopkg.lock

```diff
@@ -396,6 +402,12 @@
   revision = "3247c84500bff8d9fb6d579d800f20b3e091582c"
   version = "v1.0.0"
 
+[[projects]]
+  name = "github.com/mholt/archiver"
```

Comment:
@JoshVanL it would be great if we could not use this library. I had it before and it causes some head aches.

We are using and vendoring the docker tools already: https://github.com/jetstack/tarmak/blob/master/vendor/github.com/docker/docker/pkg/archive
 
A note the whole things has been renamed to moby https://godoc.org/github.com/moby/moby/pkg/archive

Otherwise looking good.